saritasa_invocations/alembic.py

```diff
@@ -2,12 +2,45 @@
 
 import invoke
 
-from . import _config, printing, python
+from . import _config, docker, printing, python
+
+
+def wait_for_database(context: invoke.Context) -> None:
+    """Ensure that database is up and ready to accept connections.
+
+    Function called just once during subsequent calls of alembic commands.
+
+    """
+    if hasattr(wait_for_database, "_called"):
+        return
+    docker.up(context)
+    printing.print_success("Wait for database connection")
+    config = _config.Config.from_context(context)
+    for _ in range(config.alembic.connect_attempts):
+        try:
+            # Doing it manually to avoid loop
+            python.run(
+                context,
```

Comment:
`context` contains instance of `Config` object that should define params for `run` command (like `echo` and `hide`).
Please try to overwrite them here instead of adding extra way of configuring this parameters in specific tasks

Reply:
But if we do that shouldn't we override them back? I did it in same we done for k8s commands